docs/rules/customization.md

```diff
@@ -1,12 +1,32 @@
 # Customization
 
+## Variable Tracing
+The plugin allows a limit back-tracing of variables.
+This will be used to check code like here:
+```js
+const greeting_template = `<p>Hello World!</p>`;
+// ... lots of other code in between ...
+someElemenet.innerHTML = greeting_template;
+```
+
+Currently, backtracing will only allow const and let variables that contain string literals only.
+Further assignments to these variables will also be checked for validation.
+
+**Backtracing can be disabled by setting the boolean
+option `variableTracing` to `false`.**
+
+Both values are supported and tested in CI.
+
```

Comment:
Nit, we can remove a redundant empty line here (and maybe add one empty line between line 3 and line 4.